### DISCUSSION TITLE
new osu! cookie is not vertically centered

### DISCUSSION THREAD

**Discussion by tovc** [ppy/osu#26816] [Ideas]
left is the new cookie, right is the old cookie
![image](https://github.com/ppy/osu/assets/49025619/44b6384a-a1dc-4c28-8a47-96a72e86ba2d)

the old one isn't perfectly aligned either, but it's close enough that it's not noticable by eye. Yes I'm aware that I'm mesauring the height of the lowercase text and not the exclamation mark

**Comment by LunaticaUWU** [comment]
It's made like that cause it's a graphical trick. Looks nicer to eye.

**Reply by tovc** [reply]
It looks unbalanced to me
